spec.html

```diff
@@ -25603,7 +25603,7 @@ <h1>NativeError ( _message_ )</h1>
           <p>When a _NativeError_ function is called with argument _message_, the following steps are taken:</p>
           <emu-alg>
             1. If NewTarget is *undefined*, let _newTarget_ be the active function object; else let _newTarget_ be NewTarget.
-            1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, <code>"%NativeErrorPrototype%"</code>, &laquo; [[ErrorData]] &raquo;).
+            1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, <code>"%<var>NativeError</var>Prototype%"</code>, &laquo; [[ErrorData]] &raquo;).
```

Comment:
i don't understand why the var tags belong here; `NativeErrorPrototype` is just an arbitrary string.

Same throughout this commit - perhaps it should just be removed?

Reply:
It's not an arbitrary string -- `OrdinaryCreateFromConstructor` requires it to be the name of an intrinsic object, and there's no intrinsic named `%NativeErrorPrototype%`.

Rather, within [19.5.6](https://tc39.es/ecma262/#sec-nativeerror-object-structure),
> references to _NativeError_ in the definition should be replaced with the appropriate error object name from 19.5.5

So `"%<var>NativeError</var>Prototype%"` is a stand-in for
* `"%EvalErrorPrototype%"`
* `"%RangeErrorPrototype%"`
* etc

as described at the bottom of 19.5.6.1.1. (Although now, the alg step and the note are out of sync re old-style vs new-style.)

Reply:
thanks, that makes sense